app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -307,6 +307,13 @@ DeviceDelegateOculusVR::SetRenderMode(const device::RenderMode aMode) {
   if (aMode == m.renderMode) {
     return;
   }
+  if (aMode == device::RenderMode::Immersive) {
```

Comment:
You also need to disable extra latency mode when the render mode is set back to Standalone